lib/util/exec/index.ts

```diff
@@ -65,27 +58,20 @@ function dockerEnvVars(
   );
 }
 
-export async function exec(
-  cmd: string | string[],
-  opts: ExecOptions = {}
-): Promise<ExecResult> {
-  const { env, docker, cwdFile } = opts;
-  const { binarySource, dockerChildPrefix, customEnvVariables, localDir } =
-    getAdminConfig();
-  const extraEnv = { ...opts.extraEnv, ...customEnvVariables };
-  let cwd;
-  // istanbul ignore if
-  if (cwdFile) {
-    cwd = join(localDir, dirname(cwdFile));
-  }
-  cwd = cwd || opts.cwd || localDir;
-  const childEnv = createChildEnv(env, extraEnv);
+function getCwd({ cwd, cwdFile }: ExecOptions = {}): string {
```

Comment:
What about rename `./common.ts` to `./types.ts` and move helper function to `./common.ts`, so this files gets a lot smaller ?

Reply:
I would like to perform further refactoring in separate PRs, because the rabbit hole goes so deep here